lib/node_modules/@stdlib/lapack/base/dlaset/README.md

```diff
@@ -0,0 +1,248 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2025 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dlaset
+
+> LAPACK routine to initialize an M by N matrix `A` to `beta` on the diagonals and `alpha` on the off diagonals.
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var dlaset = require( '@stdlib/lapack/base/dlaset' );
+```
+
+#### dlaset( order, uplo, M, N, A, LDA, alpha, beta )
```

Comment:
@aayush0325 I am not sure why you deviated from LAPACK here. The signature should be

```
dlaset( order, uplo, M, N, alpha, beta, A, LDA )
```

Ref: https://netlib.org/lapack/explore-html/d0/de5/group__laset_gad8051330f20413bd2a4ee0bccaf54ec8.html

This applies throughout this PR and also ensures consistency with the existing packages `lapack/base/zlaset` and `lapack/base/claset`.